MetadataExtractor/Formats/Tga/TgaDeveloperReader.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Drew Noakes and contributors. All Rights Reserved. Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using MetadataExtractor.IO;
+using System.IO;
+
+namespace MetadataExtractor.Formats.Tga
+{
+    /// <summary>Reads TGA image file extension area.</summary>
+    /// <author>Dmitry Shechtman</author>
+    sealed class TgaDeveloperReader : TgaDirectoryReader<TgaDeveloperDirectory, IndexedReader>
```

Comment:
Please use explicit accessibilities. In general these inner-working types are made public in this library.

Reply:
Should they really be public?

Reply:
Maybe. Pros and cons either way. I don't feel strongly about them being public in this instance.

For reference, often people ask how to do specific things with a format and the answer is usually to use the classes like this for their purposes. In this case they appear to be a fixed set of internal processing types, so they probably don't need to be public.